src/app/log-in/log-in.component.html

```diff
@@ -81,6 +83,10 @@ <h3 class="text--neutral-white">Log in to {{ globalVars.hostname }}</h3>
         </div>
       </li>
     </ul>
+    <div class="warning--error warning--error--medium margin-bottom--base" *ngIf="globalVars.accessLevelRequest === 4">
```

Comment:
The condition here should be `showAccessLevels && globalVars.accessLevelRequest === 4` - we don't want to show this warning for full access hosts.